src/pages/SingleVideo/SingleVideo.js

```diff
@@ -29,148 +48,169 @@ export const SingleVideo = () => {
-										Duration: {getVideoDuration(duration)}
-									</span>
-									<div className="video_card__nav_items">
-										<button
-											className="btn btn--primary btn--icon btn--round"
-											title={
-												isVideoInList(video, likes)
-													? "remove from like list"
-													: "Like"
-											}
-											onClick={toggleLikedVideo}
-										>
-											{getLikeBtn}
-										</button>
-										<button
-											className="btn btn--primary btn--icon btn--round"
-											title={
-												isVideoInList(video, watchlater)
-													? "remove from watch later"
-													: "Watch Later"
-											}
-											onClick={toggleWatchLaterVideo}
-										>
-											{getWatchLaterBtn}
-										</button>
-										<button
-											className="btn btn--primary btn--icon btn--round"
-											title="Playlist"
-										>
-											{bxIcons.playlist}
-										</button>
-									</div>
-								</section>
-							</div>
-							<h2 className="video_channel__title">{channelTitle}</h2>
+		singleVideo && (
+			<div className="home_page">
+				<main className="main--homepage single_video_page">
+					<section className="video__player_container">
+						<section className="video__player">
+							<ReactPlayer
+								url={url + videoId}
+								controls
+								onPlay={onPlayHandler}
+								playing={shouldPlay}
+								width="100%"
+								height="65vh"
+							/>
 						</section>
+						<section className="video_player__body">
+							<section className="video_player__text">
+								<div className="video__details_bar">
+									<h1 className="video_title" title={title}>
+										{title}
+									</h1>
+									<section className="video_card__nav">
+										<span className="video__duration">
+											Duration: {getVideoDuration(duration)}
+										</span>
+										<div className="video_card__nav_items">
+											<button
+												className="btn btn--primary btn--icon btn--round"
+												title={
+													isVideoInList(singleVideo, likes)
+														? "remove from like list"
+														: "Like"
+												}
+												onClick={toggleLikedVideo}
+											>
+												{getLikeBtn}
+											</button>
+											<button
+												className="btn btn--primary btn--icon btn--round"
+												title={
+													isVideoInList(singleVideo, watchlater)
+														? "remove from watch later"
+														: "Watch Later"
+												}
+												onClick={toggleWatchLaterVideo}
+											>
+												{getWatchLaterBtn}
+											</button>
+											<button
+												className="btn btn--primary btn--icon btn--round playlist_icon"
+												title="Playlist"
+												onClick={togglePlaylistPanel}
+											>
+												{bxIcons.playlist}
+											</button>
+										</div>
+									</section>
+								</div>
+								<h2 className="video_channel__title">{channelTitle}</h2>
+							</section>
```

Comment:
I guess your code is not formatted, it has too much spacings, try to use prettier

Reply:
Will look into it.